### ISSUE TITLE
py3: Try to sort the list of added .eopkgs on indexing for better output

### ISSUE THREAD

**Issue by ermo** [getsolus/eopkg#96]
... 'cause the current view is sort of confusing. It happens in parallel and prints pkg names whenever currently.

**Comment by ermo** [comment]
@sheepman4267 Added my notes above per your request.

**Comment by sheepman4267** [comment]
> Code that does indexing will need to be updated in sync with eopkg code. Hence it makes sense to keep the functionality in eopkg by default?
> 
> However, there is nothing stopping an intrepid soul from doing an actual python script that imports the eopkg modules and then massages the output?
> 
> I think the real question is "how many different code bases should implement the eopkg stuff, and how many is it feasible to keep in sync?"
> 
> To me, the obvious answer to that question (assuming the goal is to persist with eopkg-py3 for as long as needed) is "Exactly one."
> 
> The corollary to that is "add local repo support to eopkg, modelling the support of it from the way they are currently used."
> 
> But I would say that wouldn't I, given I authored the original "this is how you use a local repo to do development" documentation some years ago... 😅

I agree. And in my "handle it externally" proposal, the actual index generation would be done by `eopkg index` with its current terse output. Then, a separate script also called by go-task would analyze the diff between the previous local repo index and the new one. That way, eopkg does the necessary heavy lifting, but the more usage-specific UI case is handled externally. 

I *do* also see a good case for integrating all of this handling into eopkg itself. Just trying to explore all the options.
